package.json

```diff
@@ -8,7 +8,8 @@
   	"react": "16.13.1",
   	"react-dom": "16.13.1",
     "sass": "1.32.5",
-    "synapse-react-client": "1.15.85",
+    "sanitize-html": "^1.18.4",
```

Comment:
`sanitize-html` is omitted from the Rollup build and it's needed to sanitize annotations before we render them in HTML in the SRC EntityBadge. Currently it's only used in the EntityFinder, so it just needs to go in with it.